lib/models/applicant.rb

```diff
@@ -1,17 +1,20 @@
 class Applicant < ActiveRecord::Base
-  def self.random
-    order('RANDOM()')
-  end
 
-  def grid_id
-    # TODO
+  before_validation :compute_grid_id
```

Comment:
When importing, the records without grid IDs won't get saved. We should make a rake task that compares the import spreadsheet -- the full set of IDs -- to the IDs of the records that actually made it in.